README.md

```diff
@@ -64,6 +66,18 @@ Basic configuration options are detailled [here](https://kubernetes.github.io/in
 
 You can find your external MongoDB prerequisites [here](/doc/mongo-prerequisites.md).
 
+#### OpenShift
+
+You have a specific values-custom to ease the deployment of NeoLoad Web on OpenShift: values-custom-openshift.yaml.
+
+In this file you will have to modify the security context parameters to match your cluster rights.
+You can find documentation on security context [here](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/).
+
+For enable TLS on your routes you should:
+- Have a valide certificate.
+- Fill your certificate in the values-custom-openshift file 
+- Or reference the name of the secret that hold that certificat (by default in OpenShift you have router-certs secret in the default workspace).
```

Comment:
that hold**s** that certificat**e**

Reply:
(by default in OpenShift there is a router-certs secret in the default workspace)